packages/kbn-telemetry-tools/src/tools/serializer.ts

```diff
@@ -158,6 +165,16 @@ export function getDescriptor(node: ts.Node, program: ts.Program): Descriptor |
     if (symbolName === 'Date') {
       return { kind: TelemetryKinds.Date, type: 'Date' };
     }
+
+    // Support Array<T>
+    if (symbolName === 'Array') {
+      if (node.typeArguments?.length !== 1) {
+        throw Error('Array type only supports 1 type parameter Array<T>');
+      }
```

Comment:
UltraNIT: better safe than sorry I guess, but I still wonder if this assertion is really useful.

Reply:
It is not needed but to stay consistent with other symbols we check for.